### DISCUSSION TITLE
how to use other plugins

### DISCUSSION THREAD

**Discussion by luweizheng** [vuepress-theme-hope/vuepress-theme-hope#2760] [Q&A] [answered]
I'm not familiar with Vue. But I have a question.

Plugins usually come with the following:

```
// .vuepress/config.js
module.exports = {
  plugins: [
    '@vuepress/html-redirect', // OR full name: '@vuepress/plugin-html-redirect'
  ],
}
```

But in this theme, we should import and then use like:
```
// .vuepress/config.js
import { redirectPlugin } from "vuepress-plugin-redirect";

export default {
  plugins: [
    redirectPlugin({
    ...
    }),
  ],
};
```

So if I want to use third-party plugins, what should I do? 
First, import an instance(not sure if it should be called instance in JS) from another package and then use the instance. Just like this?

```
import { redirectPlugin } from "plugin-html-redirect";

export default {
  plugins: [
    redirectPlugin({
    ...
    }),
  ],
};
```

**Comment by Mister-Hope** [comment, marked as answer]
> But in this theme

Not in this theme, see https://v2.vuepress.vuejs.org/guide/plugin.html
